ckanext/saml2/plugin.py

```diff
@@ -401,7 +401,10 @@ def manage_permissions(self):
         try:
             logic.check_access('sysadmin', context, {})
         except logic.NotAuthorized:
-            base.abort(401, p.toolkit._('Need to be system administrator to administer'))
+            code, msg = 403, 'Not authorized to see this page'
```

Comment:
If I'm reading this correctly, it's backwards. It should be:
User not authenticated = 401 Unauthorized (will trigger SSO)
User authenticated but unauthorized = 403 Forbidden